proxy-identity/main.go

```diff
@@ -146,3 +143,12 @@ func generateAndStoreCSR(p, id string, key *ecdsa.PrivateKey) ([]byte, error) {
 
 	return csrb, nil
 }
+
+func runProxy() {
+	cmd := exec.Command("/usr/lib/linkerd/linkerd2-proxy")
+	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+	err := cmd.Run()
```

Comment:
This isn't really the same as the shell script. Specifically, I think we need to use `[syscall.Exec](https://golang.org/pkg/syscall/#Exec)` so that the proxy can actually receive signals

Reply:
I think you are technically right about them not being the same but I think by default both processes would receive signals so for the most part it would behave the same as the shell script. I guess it is pretty easy to test the signal aspect. Anyways I'm happy to make the change if that is what we want to do.